src/pkcs11/openssl.c

```diff
@@ -525,10 +524,15 @@ static CK_RV gostr3410_verify_data(const CK_BYTE_PTR pubkey, CK_ULONG pubkey_len
 			buf_len = EC_POINT_point2oct(group, P, POINT_CONVERSION_COMPRESSED, NULL, 0, NULL);
 			if (!(buf = malloc(buf_len)))
 				r = -1;
-			if (r == 1 && P)
-				r = EC_POINT_point2oct(group, P, POINT_CONVERSION_COMPRESSED, buf, buf_len, NULL);
+			if (r == 1 && P) {
+				size_t len = EC_POINT_point2oct(group, P, POINT_CONVERSION_COMPRESSED, buf, buf_len, NULL);
+				if (len == 0) {
```

Comment:
```suggestion
				if (EC_POINT_point2oct(group, P, POINT_CONVERSION_COMPRESSED, buf, buf_len, NULL) == 0) {
```

Reply:
I would prefer to use here temporary variable. At least for the reason that very long function calls in the `if` conditions format weirdly in the clang-format.